tools/setup/install-sgrep

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e
+
+if [ ! -d "/usr/local/bin/sgrep-lint-files" ]
+then
+    tmpdir="$(mktemp -d)"
+    trap 'rm -r "$tmpdir"' EXIT
+    cd "$tmpdir"
+    wget -nv "https://github.com/returntocorp/sgrep/releases/download/v0.4.9b3/sgrep-0.4.9b3-ubuntu-16.04.tgz"
+    wget -nv "https://github.com/returntocorp/sgrep/releases/download/v0.4.9b3/sgrep-0.4.9b3-ubuntu-16.04.tgz.sha256"
+    sha256sum -c "sgrep-0.4.9b3-ubuntu-16.04.tgz.sha256"
```

Comment:
Check the hash against a known value, since anyone who could manipulate the tarball could also manipulate the `.sha256` file at the same location:

```suggestion
    sha256sum -c <<EOF
7ddd8d73e7b1619d826c086950dfd1b42ac6e7cef6aebf7a3f2aa890ffcb7cfd  sgrep-0.4.9b3-ubuntu-16.04.tgz
EOF
```

To make it easier to update the script, maybe there should be shell variables like `$sgrep_version`, `$sgrep_tarball`, `$sgrep_tarball_sha256`.

Reply:
(I took my own advice in #14419 for `install-shellcheck`.)

Reply:
I'll have to probably bump to b4 to address the other concerns but will just get this suggestion in.